api/src/pcapi/routes/backoffice/filters.py

```diff
@@ -256,6 +256,18 @@ def format_booking_status_long(booking: bookings_models.Booking | educational_mo
     return '<span class="badge text-bg-success">Le jeune a réservé l\'offre</span>'
 
 
+def format_booking_validation_author_type(validationAuthorType: bookings_models.BookingValidationAuthorType) -> str:
```

Comment:
Si je suis bien les appels, cette fonction peut recevoir `None`, pour toutes les réservations `USED` passées, qui n'ont forcément pas cette colonne remplie. Ainsi le prototype peut être clarifié :
```suggestion
def format_booking_validation_author_type(validationAuthorType: bookings_models.BookingValidationAuthorType | None) -> str:
```